src/SignalR/common/Http.Connections/src/Internal/HttpConnectionContext.cs

```diff
@@ -68,6 +69,8 @@ public HttpConnectionContext(string id, ILogger logger)
             Features.Set<IHttpContextFeature>(this);
             Features.Set<IHttpTransportFeature>(this);
             Features.Set<IConnectionInherentKeepAliveFeature>(this);
+
+            _status = (int)HttpConnectionStatus.Inactive;
```

Comment:
Nit: you can keep the assignment could be inline with the declaration like before